### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_Manageyouraccounttwofactor_NonTextContrast: The Color contrast ratio of the focus indicator of the "Set up Authenticator app" and "Reset Authenticator app" button is less than 3:1.

### ISSUE THREAD

**Issue by Swati1700** [dotnet/aspnetcore#42401]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopWeb;#Win11;#ColorContrast;#Benchmark;#Chrome;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2
Chrome: Version 102.0.5005.115 (Official Build) (64-bit)

### Steps to Reproduce:

1. open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir angular"
3. move to the created directory using command "cd foldername"
4. run the following command "dotnet new angular -au Individual" and hit enter.
5. run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. browser page will open with above template loaded
9. to get access to all the pages you'll need to register first and then log-in.
10. Login with your registered account 
11. Tab till "Hello with the userid" and click on it.
12. Navigate to Two Factor Authentication and hit enter
13. Turn on Color contrast analyzer and check that The Color contrast ratio of focus indicator for "Set up Authenticator app" and "Reset Authenticator app" button is less than 3:1.

### Actual:
The Color contrast ratio of the focus indicator of the "Set up Authenticator app" and "Reset Authenticator app" button is less than 3:1.

**Similar issue also observed in the below scenarios:
Issue 1: ASP.NET Core - ManageyouraccountpersonalData**

1. open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir angular"
3. move to the created directory using command "cd foldername"
4. run the following command "dotnet new angular -au Individual" and hit enter.
5. run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. browser page will open with above template loaded
9. to get access to all the pages you'll need to register first and then log-in.
10. Login with your registered account 
11. Tab till "Hello with the userid" and click on it.
12. Navigate to Personal data and hit enter
13. Turn on Color contrast analyzer and check that The Color contrast ratio of focus indicator for "Download" and "Delete" button is less than 3:1.

### Expected:
The Color contrast ratio of the focus indicator of the "Set up Authenticator app" and "Reset Authenticator app" button should be more than or equal to 3:1.

### User Impact:
Low vision users won't be able to see the focus indicator.
![Screenshot (1145)](https://user-images.githubusercontent.com/86710888/175518186-c377032b-938d-4e69-93b9-8ef627a1dff9.png)

**Comment by HaoK** [comment]
Should be fixed by https://github.com/dotnet/spa-templates/pull/73

**Comment by Swati1700** [comment]
We downloaded the latest RC1 build from the first column of the above table and verified the bug, and the bug still reproduces.
Please find the attachment of the version.
![image](https://user-images.githubusercontent.com/86710888/185591576-8b145739-994a-4b1f-b8d3-1037011c3427.png)
Could you please provide the repro steps that you followed, if the issue is not reproducing at your end.

**Comment by merriemcgaw** [comment]
@Swati1700 - latest builds should now have it. There was a small issue with flowing into the installers, but it should be addressed now.

**Comment by Swati1700** [comment]
We have installed latest build dotnet-sdk-7.0.100-rc.2.22452.3-win-x64 but getting a below error.
![image](https://user-images.githubusercontent.com/86710888/188159269-87a3fd04-9fa8-4522-a319-7263dc9fc878.png)

**Comment by merriemcgaw** [comment]
@Swati1700 - the updated machine setup instructions are as follows:
1. On a clean machine install .NET RC2 SDK from the **Top, Middle** cell in the [Installer Table](https://github.com/dotnet/installer#table)
2. Open a command prompt and enter the following command: dotnet nuget add source -n dotnet7 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json
3. Follow the test repro steps from here.

**Comment by Swati1700** [comment]
Verified the issue and the issue is fixed. Hence closing the bug.
![Screenshot (28)](https://user-images.githubusercontent.com/86710888/192512898-f476552f-07d6-4d20-8884-a6a725230f8f.png)

**Comment by mkArtakMSFT** [comment]
Issue is fixed. Adding #Closed; tag.
